docs/guides/go-built-in.md

```diff
@@ -98,7 +98,7 @@ module github.com/me/example
 go 1.20
 
 require (
-	github.com/cometbft/cometbft v0.37.0
+github.com/cometbft/cometbft v0.38.0
```

Comment:
might be good to preserve indentation

indentation might be due to linter/formatter (go fmt)